pkg/cli/whoami/whoami.go

```diff
@@ -153,14 +164,34 @@ func (o *WhoAmIOptions) Run() error {
 		}
 		fmt.Fprintf(o.Out, "%s\n", consoleUrl)
 		return nil
-	}
+	case o.Output == "yaml":
+		u, err := o.UserInterface.Users().Get(context.TODO(), "~", metav1.GetOptions{})
```

Comment:
SelfSubjectReview needs to take precedence from UserInterface (logic https://github.com/openshift/oc/blob/093ad8719d283e1a9f79584a6c92aa9b687d7f5b/pkg/cli/whoami/whoami.go#L88-L107). Actually, there is no need to duplicate the effort. Why can't we print what we get in `WhoAmI` function.